core/controllers/certificate_assessment_test.py

```
@@ -505,3 +543,108 @@ def test_get_returns_hardcoded_attempts_list(self) -> None:
+        )
+        handler.user_id = 'user_id_1'
+        attempt = certificate_assessment_services.gae_models.CertificateAssessmentAttemptModel.create(
+            learner_id='user_id_1',
+            total_score=0.0,
+            attempt_index=1,
+            attempt_data={},
+            version_data={
+                'question_versions': {'q1': 1},
+                'question_topic_links': {'q1': ['topic_1']},
+            },
+            started_at=datetime.datetime.utcnow(),
```

Ruff (0.16.0)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)